system-tests/projects/unify-onboarding-with-config/cypress.config.js

```diff
@@ -1,18 +1,16 @@
+const { devServer } = require('@cypress/webpack-dev-server')
+
 module.exports = {
   component: {
     supportFile: false,
     testFiles: '**/*cy-spec.{js,jsx,ts,tsx}',
     componentFolder: 'src',
     devServer (cypressDevServerConfig) {
-      const { startDevServer } = require('@cypress/webpack-dev-server')
-
-      return startDevServer({
-        options: cypressDevServerConfig,
-        webpackConfig: {
-          output: {
-            publicPath: '/',
-          },
-        } })
+      return devServer(cypressDevServerConfig, {
```

Comment:
Should we move the `devServerConfig` out of the function into the `component` object, or is it done in the `devServer` function for a reason?